src/configurator/configurator-webpack-helpers.js

```diff
@@ -14,11 +14,13 @@ export function addPlugin(webpackConfig, plugin) {
 export function assignModuleRuleAndResolver(
   webpackConfig,
   rules,
-  resolverExts
+  resolverExts,
+  aliases = {}
```

Comment:
When `aliases` is empty, we should not show it at all. Currently an empty object `{}` is shown.